torchbenchmark/util/framework/vision/args.py

```diff
@@ -9,6 +9,7 @@ def parse_args(model: BenchmarkModel, extra_args: List[str]) -> argparse.Namespa
     parser.add_argument("--eval-fp16", action='store_false', help="enable eval fp16")
     parser.add_argument("--fx2trt", action='store_true', help="enable fx2trt")
     parser.add_argument("--flops", action='store_true', help="enable flops counting")
+    parser.add_argument("--train_cudagraph", action='store_true', help="enable CUDA Graph for train")
```

Comment:
Naming nit: why not simply 'cudagraph'? It can also be used for inference.

Reply:
Unfortunately for now we don't have know if the user is attempting to use inference at the point of initialization... I have been hoping to change the API to mark if it is a train or eval test, but haven't got time to do so.